### PR TITLE
Contact Form Validation

### PR DESCRIPTION
**Pull request by Whoistolu**
## Contact Form Validation

In this Milestone, we:

- Used client-side validation to catch and throw errors in the UI.
- Implemented a validation to the email field to make its content lowercase.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -329,7 +329,7 @@ <h2 class="form-h2">I'm always<br>interested in hearing<br>about new projects,<b
                     <h3 class="form-h3">I'm always interested<br>in hearing about new<br>projects, so if you'd<br>like
```
Comment (SajjadAhmad14) [inline index.html:329]: Please add unnecessary files and folders like `.vscode` to .gitignore.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Toluwase Ajise, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
- [ ] GitHub flow is followed.
- [ ] README is professional.
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Whoistolu) [reply index.html:329]: ## Thank you for your feedback @SajjadAhmad14.
The changes have been implemented.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
### Hi Toluwase Ajise, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**